packages/react-dom/src/__tests__/ReactDOMInput-test.js

```diff
@@ -1270,6 +1279,36 @@ describe('ReactDOMInput', () => {
       expect(node.getAttribute('value')).toBe('2');
     });
```

Comment:
I had to update these two tests because overriding the `value` property was done such that you couldn't check the current value.